linux/map.jinja

```diff
@@ -185,3 +185,28 @@
          },
     },
 }, grain='os_family', merge=salt['pillar.get']('linux:storage')) %}
+
+
+{% set monitoring = salt['grains.filter_by']({
```

Comment:
{%- from "linux/map.jinja" import system,network,storage,monitoring with context -%} - is it not going to blow if I get rid of map.jinja entry for this?